OpenRA.Mods.Common/Traits/Render/WithTextDecoration.cs

```diff
@@ -75,6 +75,12 @@ IEnumerable<IRenderable> IRender.Render(Actor self, WorldRenderer wr)
 			return !Info.RequiresSelection ? RenderInner(self, wr) : SpriteRenderable.None;
 		}
 
+		IEnumerable<Rectangle> IRender.ScreenBounds(Actor self, WorldRenderer wr)
```

Comment:
Not sure, just a question (I already noticed this while working on my PR): Is using `IRender` instead of `IRenderAboveShroud` in this trait intentional?
Because it uses `IRenderAboveShroudWhenSelected` when `Info.RequiresSelection` is true, so this feels a bit inconsistent (note: I haven't checked yet whether a following commit adresses this).

Reply:
This was intentional because `IRenderAboveShroudWhenSelected` is invoked directly instead of going via the screen map.  #14082 complicates matters a bit, but i'd prefer to leave the discussion and implementation to that PR -- expanding the render bounds to include very large range circles doesn't feel like a good approach to me.